documentation/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,201 @@
+## Guide for migrating to `sdk/**/arm**` from `services/**/mgmt/**`
+
+This document is intended for users that are familiar with an older version of the Azure SDK For Go for management modules (`services/**/mgmt/**`) and wish to migrate their application to the next version of Azure resource management libraries (`sdk/**/arm**`)
+
+**For users new to the Azure SDK For Go for resource management modules, please see the [README for 'sdk/armcore`](https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/armcore) and the README for every individual package.**
+
+## Table of contents
+
+* [Prerequisites](#prerequisites)
+* [General Changes](#general-changes)
+    * [Authentication](#authentication)
+    * [Error Handling](#error-handling)
+    * [Long Running Operations](#long-running-operations)
+    * [Pagination](#pagination)
+    * [Customized Policy](#customized-policy)
+    * [Custom HTTP Client](#custom-http-client)
+
+## Prerequisites
+
+Go 1.13 or above.
```

Comment:
We support the last 2 released versions of Go.  Remove the specific version reference in favor of a statement supporting N and N-1 Go releases

Reply:
I have changed this. I am not sure how to say this properly... any recommendations?